WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomerSection/CustomerSearchUICommand.swift

```diff
@@ -0,0 +1,80 @@
+import Foundation
+import Yosemite
+
+/// Implementation of `SearchUICommand` for Customer search.
+///
+final class CustomerSearchUICommand: SearchUICommand {
+
+    typealias Model = Customer
+    typealias CellViewModel = TitleAndSubtitleAndStatusTableViewCell.ViewModel
```

Comment:
Happy to get a different recommendation for the CellView we want to use here, at the moment I'm using `TitleAndSubtitleAndStatusTableViewCell` just for convenience, as must conform to `SearchResultCell` there are several options to choose from without having to create a new one.

Reply:
Given that you're going to have the avatar in there too, you might want to try adding `SearchResultCell` conformance to `LeftImageTitleSubtitleTableViewCell`?

The only thing it won't do is any attributed text in the subtitle for an underlined email as on Android... but I don't personally think that should be a blocker.